src/event-schemas/interaction-to-next-paint.json

```diff
@@ -0,0 +1,65 @@
+{
+    "$id": "billyzh.inp.report_all_changes.false",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "InteractionToNextPaintEvent",
+    "type": "object",
+    "properties": {
+        "version": {
+            "const": "1.0.0",
+            "type": "string",
+            "description": "Schema version."
+        },
+        "value": {
+            "type": "number",
+            "description": "Time from interaction to next paint"
+        },
+        "attribution": {
+            "type": "object",
+            "description": "Attributions for INP",
+            "properties": {
+                "interactionTarget": {
+                    "type": "string",
+                    "description": "A selector identifying the element that the user first interacted with as part of the frame where the INP candidate interaction occurred. If this value is an empty string, that generally means the element was removed from the DOM after the interaction."
+                },
+                "interactionTime": {
+                    "type": "number",
+                    "description": "The time when the user first interacted during the frame where the INP candidate interaction occurred (if more than one interaction occurred within the frame, only the first time is reported"
```

Comment:
fix: missing parenthesis at end
question: is this a timestamp?

Reply:
Addressed in rev2